lte/gateway/deploy/roles/ghz/tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+# Copyright 2020 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+###############################################
+# Download and build gHZ (load test dependency)
+###############################################
+- name: Clone ghz repo
+  become: yes
+  git:
+    repo: 'https://github.com/bojand/ghz'
+    dest: "{{ ghz_root }}"
+    version: v0.94.0
+    force: yes
+  when: preburn
```

Comment:
If I remember correctly, that means we have to build a new Vagrant image, provisioning wont' be enough